src/plugins/vis_type_timeseries/server/routes/post_vis_schema.ts

```diff
@@ -17,59 +17,65 @@
  * under the License.
  */
 
-import Joi from 'joi';
-const stringOptionalNullable = Joi.string()
-  .allow('', null)
-  .optional();
-const stringRequired = Joi.string()
-  .allow('')
-  .required();
-const arrayNullable = Joi.array().allow(null);
-const numberIntegerOptional = Joi.number()
-  .integer()
-  .optional();
-const numberIntegerRequired = Joi.number()
-  .integer()
-  .required();
-const numberOptional = Joi.number().optional();
-const queryObject = Joi.object({
-  language: Joi.string().allow(''),
-  query: Joi.string().allow(''),
+import { schema } from '@kbn/config-schema';
+import { TypeOptions } from '@kbn/config-schema/target/types/types';
+
+const stringOptionalNullable = schema.nullable(schema.string());
```

Comment:
Looks like this should be `schema.maybe(schema.nullable(schema.string()));`

The current schema just gives `string | null`.

Reply:
`README.md` from `kbn-config-schema` says:
![image](https://user-images.githubusercontent.com/54894989/81310714-2132fe80-908d-11ea-8d94-2b98ffa3843d.png)